schema/spacewalk/postgres/tables/rhnPackageCapability_index.sql

```diff
@@ -13,13 +13,13 @@
 -- in this software or its documentation.
 --
 
-CREATE UNIQUE INDEX rhn_pkg_cap_name_version_uq
-    ON rhnPackageCapability (name, version)
- where version is not null;
+CREATE INDEX rhn_pkg_cap_name_version_uq
```

Comment:
What is the reason for the removal of the `UNIQUE` clause?

Reply:
It was done by accident, will put it back. Thanks for noticing it.

Reply:
Put `UNIQUE` back with https://github.com/uyuni-project/uyuni/pull/4073/commits/3902deae64cb7e9c4a0b244f65add7103b4f299c